airbyte-integrations/connectors/source-us-census/README.md

```diff
@@ -1,45 +1,52 @@
-# Us-Census source connector
+# Us Census Source
 
-
-This is the repository for the Us-Census source connector, written in Python.
+This is the repository for the Us Census configuration based source connector.
 For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.com/integrations/sources/us-census).
 
 ## Local development
 
 ### Prerequisites
-* Python (~=3.9)
-* Poetry (~=1.7) - installation instructions [here](https://python-poetry.org/docs/#installation)
+
+* Python (`^3.9`)
```

Comment:
This should be 3.10 really, but this would be cleaned out by manifest-only migration anyway!